packages/mdc-select/component.ts

```diff
@@ -153,11 +153,12 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
     this.handleKeydown_ = (evt) => this.foundation_.handleKeydown(evt);
     this.handleMenuSelected_ = (evtData) => this.selectedIndex = evtData.detail.index;
     this.handleMenuOpened_ = () => {
-      // Menu should open to the last selected element.
-      if (this.selectedIndex >= 0) {
-        const selectedItemEl = this.menu_!.items[this.selectedIndex] as HTMLElement;
-        selectedItemEl.focus();
-      }
+      if (this.menu_!.items.length === 0) return;
+
+      // Menu should open to the last selected element, should open to first menu item otherwise.
+      const selectedItemIndex = this.selectedIndex >= 0 ? this.selectedIndex : 0;
```

Comment:
Maybe rename these variables to e.g. `focusedItemIndex` and `focusedItemEl` since it's not always the selected item (if there is none)?